app/inpututils.cpp

```diff
@@ -1970,3 +1971,8 @@ void InputUtils::updateFeature( const FeatureLayerPair &pair )
   pair.layer()->commitChanges();
   pair.layer()->triggerRepaint();
 }
+
+QString InputUtils::imageGalleryLocation()
+{
+  return QStandardPaths::standardLocations( QStandardPaths::PicturesLocation ).last();
```

Comment:
In theory standardLocations() can also return empty list, so it may be worth adding a check and emit a warning + return empty string if no path is available...